2022/09/22 17:35:54 - Terminal_Log - INFO      - log_save.py : 22 line - run the section python_env_step
2022/09/22 17:35:54 - Terminal_Log - INFO      - log_save.py : 22 line - run commnad: set http_proxy= http://proxy-iind.intel.com:911;https://proxy-iind.intel.com:911; dir
2022/09/22 17:35:54 - Terminal_Log - INFO      - log_save.py : 22 line - out:  

2022/09/22 17:35:54 - Terminal_Log - ERROR     - log_save.py : 28 line - err:  

